Dockerfile

```diff
@@ -113,6 +121,9 @@ COPY --from=base /rootfs ./
 WORKDIR /src
 COPY hack/scripts/cleanup.sh /bin
 RUN cleanup.sh /initramfs
+# bootloader
+COPY --from=base /rootfs/share/gptmbr.bin /initramfs/usr/share/gptmbr.bin
+COPY --from=base /rootfs/bin/extlinux /initramfs/bin/extlinux
```

Comment:
I think we can drop this line. We already do a `COPY --from=base /rootfs ./` a few steps before.

Reply:
We need to have these here to add them back in after the cleanup script runs. Seemed like the simpler workaround instead of updating the cleanup.sh to exclude these files.

Reply:
Agree we need to keep `COPY --from=base /rootfs/share/gptmbr.bin /initramfs/usr/share/gptmbr.bin`.
But looking at the cleanup script, we can drop `COPY --from=base /rootfs/bin/extlinux /initramfs/bin/extlinux`